### ISSUE TITLE
Puppeteer Library BUG: this.browser.deleteCookie is not a function

### ISSUE THREAD

**Issue by elena-dev-thought** [Codeception/CodeceptJS#954]
#### What are you trying to achieve?
Run tests locally using Puppeteer 

#### What do you get instead?
Defects:
     this.browser.deleteCookie is not a function

> Provide console output if related. Use `--verbose` mode for more details.

```bash
     this.browser.deleteCookie is not a function
  ypeError: this.browser.deleteCookie is not a function
      at Puppeteer._after (node_modules/codeceptjs/lib/helper/Puppeteer.js:133:26)
      at recorder.add (node_modules/codeceptjs/lib/listener/helpers.js:21:69)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)

```

> Provide test source code if related

The test is working with WebdriverIO, but not with puppeteer
```js
// paste test
```

### Details

* CodeceptJS version: 1.1.5
* NodeJS Version: 
* Operating System: OS High Sierra 
* Protractor || WebDriverIO || Nightmare version (if related): Puppeteer 1.0.0
* Configuration file:

```js
# paste suite config here
```

**Comment by reubenmiller** [comment]
I've confirmed the bug, I'll submit a PR very soon to fix it.

**Comment by reubenmiller** [comment]
Fixed in v1.1.6 which was release today.
